homework_6/pandas.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Задание 1. Работа с реальными данным
+
+# In[448]:
+
+
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+
+# In[449]:
+
+
+def read_gff(path):
+    c_names = ['seq_id', 'source', 'type',
+                'start', 'end', 'score', 'strand',
+                'phase', 'attributes']
+    gff_df = pd.read_csv(path, names=c_names, skiprows=1, sep='\t')
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на много строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    gff_df = pd.read_csv(path, names=c_names, comment="#", sep='\t')
```